### ISSUE TITLE
No callback in RP IVR on 22nd Feb

### ISSUE THREAD

**Issue by vivek-kumar-pandey** [DostEducation/RP_IVR_analytics#350]
**Describe the bug**
We found that we are not getting any callback from RP IVR on 22nd Feb

**Add diagnosis details**
- [x] To be added

**Database query used**
NA

**To Reproduce**
- Query the table for outbound calls on 22nd feb 2023

**Expected behavior**
- All calls should go correctly

**Screenshots**
![image](https://user-images.githubusercontent.com/49094385/220640499-35703bb8-5122-40b5-9a8c-d1bee361129c.png)

**Additional context**
NA

**Comment by vivek-kumar-pandey** [comment]
We found that we are not getting the callback for the missedcalls and in rapid pro it gets queued. It happened on 22nd Feb.
On Diagnosing further we found that, 
- The last call that had gone was around 4:10 PM on 21st Feb. After that there are no outbound calls. 
- The regular calls, as well as callback for missed calls, are getting queued as in the screenshot and later it gets failed. 
- This was only happening for the UP system number.
- The impact for this is given in this screenshot. We can see for other numbers we have both inbound and outbound calls fine. But for UP number ie. ****703 there is only inbound call and no outbound call.
- We have raised this with unicef team

The issue was at the Rapid Pro or Kookoo end
